kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/UpdateTopicConfigurationRequest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.entities.v3;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import java.util.StringJoiner;
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Request body for {@code PUT /v3/clusters/<clusterId>/topics/<topicName>/configurations/<name>}
+ * requests.
```

Comment:
i don't like how much fluffy bloat there is in the request body `{ "data": { "attributes": { "value": "new_value" } } }`. i doubt the benefits outweigh the downsides - but i know this has been discussed at length.